path.go

```diff
@@ -14,12 +14,8 @@
 
 package zetcd
 
-import (
-	"path"
-)
-
 func mkPath(zkPath string) string {
-	p := path.Clean(zkPath)
```

Comment:
The detection should probably be a special case if `FlagSequence` is set on Create.